core/errors/infer.cc

```diff
@@ -23,6 +23,12 @@ ErrorClass errorClassForUntyped(const GlobalState &gs, FileRef file) {
         prodHistogramInc("untyped.usages", file.id());
     }
 
+    prodCounterInc("types.input.untyped.usages");
```

Comment:
So I think we want this at the top of the method, before the `gs.trackUntyped`.

That way, it works like how the `types.input.sends.typed`/`types.input.sends.total` stats work right now, where we can report the codebase-wide count without special flags, but if we want to break it down by file or by blame, then there are more hoops.